content/docs/user-guide/basic-concepts/stage.md

```diff
@@ -0,0 +1,24 @@
+---
+name: 'Stage'
+match: ['stage', 'stages']
+tooltip: >-
+  A stage is a single unit of execution in a pipeline that produces some
+  (intermediate or final) output. [ð](/doc/user-guide/concepts/stage)
+---
+
+# Stage
+
+A stage is a single unit of execution in a <abbr>pipeline</abbr> that takes some
```

Comment:
Should we go a bit more informal here? To be extreme: "Stage is any command that you would usually run from CLI that usually reads some files and produces other files or models. It can be something as simple as `python train.py " ...

We want here to give people sense about these things, I think. Using regular language, simple examples. Something that would make it very easy to catch the idea ...

Reply:
I think 1ea2da3 is better in that regard. Thank you.